src/unix/core.c

```diff
@@ -35,7 +35,10 @@
 #include <sys/un.h>
 #include <netinet/in.h>
 #include <arpa/inet.h>
-#include <limits.h> /* INT_MAX, PATH_MAX */
+#ifndef __need_IOV_MAX
+#define __need_IOV_MAX
+#endif
```

Comment:
I forgot about _SC_IOV_MAX, I will remove those lines.
However I still think it's better to use libc's IOV_MAX than an internal hardcoded value.